.ci/prepare_release

```diff
@@ -2,40 +2,38 @@
 
 set -e
 
-if [ -z "$SOURCE_PATH" ]; then
-  export SOURCE_PATH="$(readlink -f $(dirname ${0})/..)"
+if [ -z "$REPO_DIR" ]; then
+  export REPO_DIR="$(readlink -f $(dirname ${0})/..)"
 else
-  export SOURCE_PATH="$(readlink -f "${SOURCE_PATH}")"
+  export REPO_DIR="$(readlink -f "${REPO_DIR}")"
 fi
 
 # not every environment has pushd
 old_pwd=$PWD
-cd "${SOURCE_PATH}"
-
-version="$(cat "${SOURCE_PATH}/VERSION")"
-if [ -z "$version" ]
-then
-  echo "Version not found"
-  exit 1
+cd "${REPO_DIR}"
+
+if [ -z "$EFFECTIVE_VERSION" ]; then
+    version="$(cat "${REPO_DIR}/VERSION")"
+    if [ -z "$version" ]
+    then
+      echo "Version not found"
+      exit 1
+    fi
+else
+    version="${EFFECTIVE_VERSION}"
 fi
 
-# revert changes VERSION
-git checkout "${SOURCE_PATH}/VERSION"
+# revert changed VERSION
+git checkout "${REPO_DIR}/VERSION"
```

Comment:
btw: the patching of VERSION file within work tree can be disabled via `pipeline_definition`; then there would not be the need to revert it